assemble/bin/accumulo-service

```diff
@@ -159,10 +210,13 @@ function main() {
           start_service "${@:3}"
           ;;
         stop)
-          stop_service
+          stop_service "${@:3:1}"
           ;;
         kill)
-          kill_service
+          kill_service "${@:3:1}"
```

Comment:
I agree that shift is better so I can look at modifying this code to avoid this. 

I need to go back and revisit the logic but iirc I was using the substring Expansion to pull the service name and ignore any additional details only on stop and kill commands. 

This was because we need the additional compaction queue or scan server group information to be passed on startup so that `accumulo-env.sh` could support assigning different JAVA_OPTS based on the scan server group or compaction queue vs just a single server type.

However `stop_service` and `kill_service` don't use those extra args for anything and instead attempt to treat them as additional service names which generate a lot of extra log message noise.

Reply:
Removed all of this in favor of local variables and passing vars between the bash functions.